lib/dependencies/RequireContextPlugin.js

```diff
@@ -34,6 +36,14 @@ class RequireContextPlugin {
 					RequireContextDependency,
 					new RequireContextDependency.Template()
 				);
+				compilation.dependencyFactories.set(
+					ImportMetaContextDependency,
+					contextModuleFactory
+				);
+				compilation.dependencyTemplates.set(
+					ImportMetaContextDependency,
+					new ImportMetaContextDependency.Template()
+				);
```

Comment:
`RequireContextPlugin.js` sounds a bit like the wrong plugin name.

Best split that into a separate `ImportMetaContextPlugin.js`